readme.md

```diff
@@ -47,6 +47,10 @@ It will try to code sign the DMG, but the DMG is still created and fine even if
 
 <img src="screenshot-dmg.png" width="772">
 
+### DMG Icon
+
+[GraphicsMagick](http://www.graphicsmagick.org/) is required to create the DMG icon based on the application icon and macOS mounted device icon.
```

Comment:
You should include how to easily get it:

```
brew install imagemagick
brew install graphicsmagick
```

And include an example screenshot on how it will look like.

Reply:
Sure. Do you mean an screenshot of how output icon looks like?

Reply:
Yes

Reply:
With the normal app icon and then the disk image icon. So they can compare.